### PR TITLE
wonton: new version

### PR DESCRIPTION
**Pull request by Tracy-Pantleo**

### CODE REVIEW

**Review by raovgarimella — DISMISSED** [review]

**Review by raovgarimella — DISMISSED** [review]

**Comment by raovgarimella** [comment]
@sethrj @scheibelp 
Somehow this didn't get categorized and assigned to a spack developer for review. Could you please review?

File: var/spack/repos/builtin/packages/wonton/package.py [hunk 1]
```diff
@@ -49,15 +50,14 @@ class Wonton(CMakePackage):
     depends_on('mpi', when='+mpi')
     depends_on('flecsi', when='+flecsi')
 
+    depends_on('jali@1.1.6', when='wonton@3.2.1: +jali')
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/wonton/package.py]: There is no `wonton@3.2.1` that I know of - was this a typo (maybe `wonton@1.3.2:` was the intent)?

Also, I think it works fine to say `when=<this-package-name>...` but generally for constraints on "this" package, you can leave off the package name.

Comment (Tracy-Pantleo) [reply var/spack/repos/builtin/packages/wonton/package.py]: Yes it was a typo thank you

File: var/spack/repos/builtin/packages/jali/package.py [hunk 2]
```diff
@@ -18,7 +18,7 @@ class Jali(CMakePackage):
     maintainers = ['raovgarimella']
 
     version('master', branch='master')
-    version('1.1.6', sha256='1321c0ca847bb74f760398a25eeb640d09f87764d3527737d9b84d7ce601d163')
+    version('1.1.6', sha256='a2f4e4f238c60ea78486e0c9ea5b3e2cdd9d91c2ae5ea006a1d33a12e9eafa3a')
```
Comment (sethrj) [inline var/spack/repos/builtin/packages/jali/package.py:21]: Why did this sha change? It's very bad from a reproducibility and security standpoint. @raovgarimella did the contents change, the distribution method, ...?

**Review by sethrj — DISMISSED** [review]

File: var/spack/repos/builtin/packages/jali/package.py (hunk 2 above)
Comment (Tracy-Pantleo) [reply var/spack/repos/builtin/packages/jali/package.py:21]: When trying to build this error message was given "expected 1321c0ca... but got a2f4e4f....."

Comment (raovgarimella) [reply var/spack/repos/builtin/packages/jali/package.py:21]: @sethrj It was my fault. I did a manual unzip and tarring and must have gotten the sha sum wrong. It went undetected until @Tracy-Pantleo tested it.

**Review by scheibelp — APPROVED** [review]

**Comment by scheibelp** [comment]
Thanks!

**Pull request merged**